File: app/templates/server_node_typescript/api/todo/routes/todo-routes.ts
"use strict";

import * as express from "express";
import {TodoController} from "../controller/todo-controller";

export class TodoRoutes {
    static init(router: express.Router) {
      router
        .route("/api/todos")
        .get(TodoController.getAll)
        .post(TodoController.createTodo);

      router
        .route("/api/todos/:id")
        .delete(TodoController.deleteTodo);
    }
}

File: app/templates/server_go/api/todo/routes/todoroutes.go
package todoroutes

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/todo/controller"
	"github.com/labstack/echo"
)

func Init(e *echo.Echo) {
	e.Get("/api/todos", todocontroller.GetAll)
	e.Post("/api/todos", todocontroller.NewTodo)
	e.Delete("/api/todos/:id", todocontroller.RemoveTodo)
}
